run.py

```diff
@@ -85,38 +209,31 @@ def parse_args_and_settings(settings_file='settings.yaml'):
 
     # parse command-line args
     parser = argparse.ArgumentParser(add_help=False)
-    parser.add_argument(
-        '-v', '--verbose', action='store_true', help='print docker stdout')
-    parser.add_argument(
-        '-p', '--pull_latest', action='store_true',
-        help='pull latest docker images before running')
-    parser.add_argument(
-        "-h", "--household_sample_size", action="store",
-        help="household sample size")
-    parser.add_argument(
-        "-s", "--static_skims", action="store_true",
-        help="bypass traffic assignment, use same skims for every run.")
+    parser.add_argument( '-v', '--verbose', action='store_true', help='print docker stdout')
+    parser.add_argument( '-p', '--pull_latest', action='store_true', help='pull latest docker images before running')
+    parser.add_argument("--data-dir", action="store", help="directory where data files are located")
+    parser.add_argument("-h", "--household_sample_size", action="store", help="household sample size")
+    parser.add_argument("-s", "--static_skims", action="store_true",
+                        help="bypass traffic assignment, use same skims for every run.")
     parser.add_argument(
         "-w", "--warm_start_skims", action="store_true",
         help="generate full activity plans for the base year only.")
-    parser.add_argument(
-        '-f', '--figures', action='store_true',
-        help='outputs validation figures')
+    parser.add_argument('-f', '--figures', action='store_true', help='outputs validation figures')
     parser.add_argument(
         '-d', '--disable_model', action='store',
         help=(
             '"l" for land use, "a" for activity demand, '
             '"t" for traffic assignment. Can specify multiple (e.g. "at")'))
-    parser.add_argument(
-        '-c', '--config', action='store',
-        help='config file name')
+    parser.add_argument('-c', '--config', action='store', help='config file name')
     args = parser.parse_args()
-    
-    if args.config:
-        settings_file = args.config
 
-    # read settings from config file
-    with open(settings_file) as file:
+    if not args.config:
+        raise RuntimeError("Please specify the settings/config file to use (--config)")
+    if not args.data_dir:
```

Comment:
Is this in case the data folder is outside the `pilates` folder? I would leave it as optional if possible.

Reply:
@jdcaicedo251 - the main reason for this change was to better support a clean separation between "my code is here" and "my data is over here". This means that we can have one code checkout and run it against many different scenarios. 

My personal opinion is that while defaults can make life easier for a user - they will also sometimes paper over an error that the user makes and result in a lot of wasted effort. This is why I quite often (as here) design around requiring the user to make their intention clear. 

In this case it would probably be okay to not require both of those to be defined - maybe require that a data directory is specified and if there isn't a config file specified it defaults to `{data_dir}/settings.yaml`

Reply:
Of course, if you feel strongly that it should be default to assuming that the src directory is the data directory and that it will contain a settings.yaml that's fine by me.

Reply:
@jdcaicedo251 let me know when you get a chance to check my replies and I'll get this thing finalised.

Reply:
@jamiecook thanks. This actually make sense for scenario creation. Thank you for the change.